solr/core/src/java/org/apache/solr/util/ThreadCpuTimer.java

```diff
@@ -87,23 +87,27 @@ public long getStartCpuTimeNs() {
    *
    * @return current value, or {@link #UNSUPPORTED} if not supported.
    */
-  public long getCurrentCpuTimeNs() {
+  public long getElapsedTimerCpuNs() {
```

Comment:
We're getting the elapsed CPU time, not a timer ... so a better name would be `getElapsedCpuTimeNs()`